_posts/2020-09-04-meet-the-team.md

```diff
@@ -0,0 +1,177 @@
+
+## Alex - [@axic](https://github.com/axic)
+
+### What is your professional background?
+
+I have a long-standing interest in open source. Before my "professional career" I have been an early contributor to MPlayer and [FFmpeg](https://ffmpeg.org), and thus my first paid roles also revolved around multimedia. Video encoding and streaming, mostly on embedded resource-constrained devices, to be precise. At some point I made a switch to financial software (payment systems) for a good number of years, unfortunately in a closed-source setting. This was a good background for working on Bitcoin startups before finally arriving at Ethereum late 2015 after Frontier was launched.
+
+### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+I keep myself busy mostly with design discussions and helping out on management related work as a *Co-Lead*. When time allows and I get the opportunity to code, then I usually find myself working on Yul or lately on the SMTChecker.
+
+Since I am involved with other parts of Ethereum, I try to bridge the gap between other technical teams and Solidity.
+
+### What would you like to see in Solidity in the future?
+
+Short-term: Compiling the Eth2 deposit contract, Uniswap V2, and Gnosis Safe with the new Yul IR, and to being able to "verify" them with the SMTChecker.
+
+Long-term: It would be nice for the language to move on from the inheritance centered design. I really like some aspects of Rust after working with it, especially traits, enums, and partially the module system. Late last year I shared some example code of a hybrid Rust-Solidity language, but it would not be wise to blatantly follow Rust, considering our special requirements. We already have plenty of discussions regarding these topics, so I am very hopeful.
+
+## Bhargava - [@bshastry](https://github.com/bshastry)
+
+### What is your professional background?
+
+*I graduated from Technische Universitaet Berlin with a Ph.D. in computer science. My Ph.D. thesis focused on developing techniques to find security vulnerabilities quicker in open-source software.*
+
+### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+*In the Solidity team, my role is to help the team find bugs and security issues as soon as possible, preferably before an upcoming release. More specifically, I develop semantic fuzzers for different compiler sub-systems e.g., compiler frontend, optimiser etc.*
+
+### What do you like to do in your free time?
+
+*I enjoy finding security issues with popular open-source software implementations and blogging about my experiences on my personal blog over at https://bshastry.github.io/blog. I also enjoy tapping the closest piece of furniture to the beat of electronic music that is being played.*
+
+## Christian - [@christianparpart](https://github.com/christianparpart)
```

Comment:
@franzihei @christianparpart what will happen to this entry?

Reply:
This probably has to be deleted for now :( (I can add his entries later if he wants).